platform/darwin/src/MGLShape.mm

```diff
@@ -94,9 +94,7 @@ - (BOOL)isEqual:(id)other
 
 - (NSUInteger)hash
 {
-    NSUInteger hash;
-    hash += _title.hash;
-    hash += _subtitle.hash;
+    NSUInteger hash = _title.hash + _subtitle.hash;
```

Comment:
his is a fairly conventional implementation, as far as I know. The only hard requirement for a `-hash` implementation is that two equal objects (as in `-isEqual:`) must have the same hash. But feel free to improve on the implementation in a separate PR.